lms/djangoapps/courseware/access.py

```diff
@@ -304,6 +330,7 @@ def _has_access_course(user, action, courselike):
 
     Valid actions:
 
+    'cms' -- can load and has access to cms view of course
```

Comment:
I don't see any code changes in this function, is this accurate?

Reply:
Sorry I meant to delete this line instead of uncommenting it.
```suggestion
```